kibble/models/config.go

```diff
@@ -33,6 +33,7 @@ type Config struct {
 	DefaultTimeZone           string                    `json:"defaultTimeZone"`
 	DefaultDateFormat         string                    `json:"defaultDateFormat"`
 	DefaultTimeFormat         string                    `json:"defaultTimeFormat"`
+	RemoteTranslations 	  bool			    `json:"remoteTranslations"`
```

Comment:
Not a fan of the wording - can we word as yes/no statement?

```suggestion
	useTranslationsApi 	  bool			    `json:"useTranslationsApi"`
```